routes/web.php

```diff
@@ -3,9 +3,15 @@
 use App\Models\Recipe;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Route;
+use Illuminate\Support\Collection;
 
 Route::get('/', function () {
-    return view('home');
+    $emojis = ['🥦', '🥕', '🌽', '🍅', '🍆', '🥔', '🥬', '🥒', '🌶', '🍠', '🥑', '🍄'];
```

Comment:
😹👌

Reply:
This cause RecipeTest to fail cause the view ws miss the randomEmoji